### PR TITLE
11252 - bcol registering party code

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#11252

*Description of changes:*
hard code bcol registering party code

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/utils/auth-helper.ts [hunk 1]
```diff
@@ -41,65 +41,68 @@ export function getKeycloakRoles (): Array<string> {
   throw new Error('Error getting Keycloak roles')
 }
 
+export async function getStaffegisteringParty (isBcOnline: boolean): Promise<PartyIF> {
+  let partyCode = '99990008'
```
Comment (kialj876) [inline ppr-ui/src/utils/auth-helper.ts]: Can you add these codes as config values? That way if they're ever changed in dev/test/prod we don't need to rerelease. Once you add them into the code I can update 1pass to include them in the deploy

**Review by doug-lovett — APPROVED** [review]
Looks good, I like Kial's suggestion.

File: ppr-ui/src/utils/auth-helper.ts (hunk 1 above)
Comment (jordiwes) [reply ppr-ui/src/utils/auth-helper.ts]: done, let me know if it makes sense

File: ppr-ui/src/utils/auth-helper.ts [hunk 2]
```diff
@@ -41,65 +41,68 @@ export function getKeycloakRoles (): Array<string> {
   throw new Error('Error getting Keycloak roles')
 }
 
+export async function getStaffegisteringParty (isBcOnline: boolean): Promise<PartyIF> {
+  let partyCode = sessionStorage.getItem(SessionStorageKeys.PprStaffPartyCode)
```
Comment (kialj876) [inline ppr-ui/src/utils/auth-helper.ts]: SessionStorageKeys is just for the common components (it doesn't include our own config stuff that we put in session storage). I don't think we have an enum or anything for it so you just need to do the string *PPR_STAFF_PARTY_CODE*

Same for below

File: ppr-ui/src/utils/auth-helper.ts (hunk 1 above)
Comment (kialj876) [reply ppr-ui/src/utils/auth-helper.ts]: yeah looks good, you just need to update how you get the session storage value which I mentioned below

**Comment by kialj876** [comment]
fyi I updated the config in 1pass to include PPR_STAFF_PARTY_CODE + BCOL_STAFF_PARTY_CODE in dev/test/prod, let me know if the code for those changes in any of the envs

File: ppr-ui/src/utils/auth-helper.ts (hunk 2 above)
Comment (jordiwes) [reply ppr-ui/src/utils/auth-helper.ts]: gotcha, made the change

**Review by kialj876 — APPROVED** [review]
thanks for the changes!

**Pull request merged**